build-system/pr-check/module-build.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
For parity with `build-system/pr-check/single-pass-tests.js`, let's call this file `build-system/pr-check/module-build-tests.js`

Reply:
Resolved, since this doesn't run any tests, and mirrors the existing `build.js`.